operations.json

```diff
@@ -0,0 +1 @@
+{"src": "c1", "dest": "n1", "body": {"msg_id": 1,          "type": "init",          "node_id": "n1",          "node_ids": ["n1"]}}
```

Comment:
Did you mean to commit this?

Reply:
This was meant more as a way to locally test my work. I can remove it.